upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -649,7 +648,7 @@ func (c *ApplyClusterCmd) Run() error {
 			BootstrapScript: bootstrapScriptBuilder,
 			Lifecycle:       clusterLifecycle,
 		})
-
+	case kops.CloudProviderDO:
```

Comment:
Might be good to have a `// Digital Ocean tasks will go here` comment here, just so we don't think this is a fall-through